zebra-rpc/src/methods.rs

```diff
@@ -657,6 +704,13 @@ pub struct GetBlockChainInfo {
     consensus: TipConsensusBranch,
 }
 
+/// The transparent balance of a set of addresses.
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, serde::Serialize)]
+pub struct AddressBalance {
+    #[serde(serialize_with = "Amount::serialize_as_string")]
```

Comment:
Question: Should this be a number?

`lightwalletd` expects a number, so this might be a docs bug:
https://github.com/zcash/lightwalletd/blob/bdaac63f3ee0dbef62bde04f6817a9f90d483b00/common/common.go#L128-L130

Reply:
Fixed, thanks for this!